core/frontend/src/utils/deviceid_decoder.ts

```diff
@@ -162,5 +163,6 @@ export default function decode(device: string, devid: number): deviceId {
     bus,
     address: toHex(address),
     devtype,
+    paramValue: devid,
```

Comment:
isn't that the same as param.value ?

Reply:
param here is a string  :sweat_smile:

Reply:
![facepalm-really](https://user-images.githubusercontent.com/1215497/217938291-3f0e7541-1822-465f-b41f-aa4802bd854d.gif)